packages/react-components/src/lib/config/index.ts

```diff
@@ -1,7 +1,9 @@
 import { assets, chains } from 'chain-registry';
-import type { Chain, AssetList } from '@chain-registry/types';
 import type { Endpoints } from '@cosmos-kit/core';
 
+type Chain = (typeof chains)[number];
+type AssetList = (typeof assets)[number];
```

Comment:
import { AssetList, Chain } from '@chain-registry/types';

Reply:
I was seeing some type mismatch and resorted to this, but tried it again and it seems to build 👍